docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
       - data-dir-conf
     volumes:
       - geoserver-data-dir:/geoserver_data/data
+      - backup-restore:/backup-restore
```

Comment:
Please use underscore in file paths rather then dash

Reply:
done